bigchaindb/core.py

```diff
@@ -136,11 +138,54 @@ def write_transaction(self, signed_transaction, durability='soft'):
         signed_transaction = deepcopy(signed_transaction)
         # update the transaction
         signed_transaction.update({'assignee': assignee})
+        signed_transaction.update({'assignment_timestamp': time()})
```

Comment:
Don't think it will work -- for stale transactions, we need to look within a window of time.  A string time stamp may not order correctly (e.g. "100" comes before "20").  For instance, you can't call RethinkDBs `between` function on "0" and "140000000" and get the right answer, I believe.